cypress/support/fragments/data_import/fileDetails.js

```diff
@@ -0,0 +1,27 @@
+import { MultiColumnListCell, Section, including, KeyValue } from '../../../../interactors';
+import invoices from '../invoices/invoices';
+import TopMenu from '../topMenu';
+
+const expectedInvoiceDate = '10/15/2021';
+const expectedInvoiceStatus = 'Open';
+const expectedInvoiceSource = 'EDI';
+
+export default {
+  checkInvoiceDetails:() => {
+    cy.do(Section().find(MultiColumnListCell(including('49072'))).perform(element => {
```

Comment:
Are you sure about this hardcoded value? Will it work in another environment?

`including('49072')`

Reply:
I put the value in a variable.